cedar-policy-generators/Cargo.toml

```diff
@@ -5,9 +5,7 @@ version = "4.0.0"
 publish = false
 
 [dependencies]
-# We had to fix the version of `arbitrary` as well as `derive_arbitrary` because 1.4.x introduces a breaking change
-arbitrary = "=1.3.2"
-derive_arbitrary = "=1.3.2"
+arbitrary = "1.3"
```

Comment:
should we put "1.4" here to guarantee we get at least 1.4?

Reply:
Done.